install-dev/controllers/http/process.php

```diff
@@ -108,7 +108,7 @@ public function process()
                 $this->processPostInstall();
             }
         } catch (\Exception $e) {
-            $this->ajaxJsonAnswer(false, $e->getMessage());
+            $this->ajaxJsonAnswer(false, (_PS_MODE_DEV_) ? (string) $e : $e->getMessage());
```

Comment:
@eternoendless why debug mode only ?

Reply:
I don't know, I'm  do we want to show a stack trace to possibly nontechnical people?
On the other hand, it could help trace errors in production... wdyt @PrestaShop/product-team ?

Reply:
@eternoendless maybe we could log it to the file when dev mode is not enabled? just like `autoupgrade`, then QA can ask people for the file if they're reporting anything related to the installation process